### ISSUE TITLE
 softmax processing on Grad CAM

### ISSUE THREAD

**Issue by zbb1111** [linyq2117/CLIP-ES#16]
Thank you for your excellent article! I would like to ask if you mentioned in the article that you performed softmax processing on Grad CAM, but I couldn't find this code. Can you provide the relevant code? Looking forward to your reply!

**Comment by linyq2117** [comment]
Thanks for your interest. We perform softmax on the classification logits [here](https://github.com/linyq2117/CLIP-ES/blob/d0d0c07b73b44c66a13dda5cdbe9f751fddb6b05/clip/model.py#L420).
